content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -184,33 +214,35 @@ A bundle can be included as an entry in multiple `olm.channel` blobs, but it can
 Also, it is valid for an entry's replaces value to reference another bundle name that cannot be found in this catalog
 (or even another catalog) as long as other channel invariants still hold (e.g. a channel cannot have multiple heads).
 
-The `olm.channel` [cue][cuelang-spec] schema is:
-```cue
-#Channel: {
-  schema: "olm.channel"
-  package: string & !=""
-  name: string & !=""
-  entries: [...#ChannelEntry]
-}
-
-#ChannelEntry: {
-  // name is required. It is the name of an `olm.bundle` that
-  // is present in the channel.
-  name: string & !=""
-
-  // replaces is optional. It is the name of bundle that is replaced
-  // by this entry. It does not have to be present in the entry list.
-  replaces?: string & !=""
-
-  // skips is optional. It is a list of bundle names that are skipped by
-  // this entry. The skipped bundles do not have to be present in the
-  // entry list.
-  skips?: [...string & !=""]
-
-  // skipRange is optional. It is the semver range of bundle versions
-  // that are skipped by this entry.
-  skipRange?: string & !=""
-}
+Here is an example of an `olm.channel` blob:
+```yaml
+schema: olm.channel
+package: foo
+name: candidate-v0
+entries:
+- # name is required. It is the name of an `olm.bundle` that
+  # is present in the channel.  
+  name: foo.v0.3.1
```

Comment:
Nit: formatting here with the comment looks a bit off, maybe something like this would look a bit cleaner?
```suggestion
 # name is required. It is the name of an `olm.bundle` that
 # is present in the channel.  
- name: foo.v0.3.1
```

Reply:
I tried moving the list `-` character to the line with `name` and ran the yaml through `yq`. The result was that `yq` outdented the `name` comment, which made it look like it went with the entire entry, not just the name.

```yaml
entries:
  # name is required. It is the name of an `olm.bundle` that
  # is present in the channel.  
  - name: foo.v0.3.1
    # replaces is optional. It is the name of the bundle that is replaced
    # by this entry. It must be present in the entry list, unless this
    # entry is the channel tail. Channel tails are allowed to have replaces
    # values that are not present in the entry list.
    replaces: foo.v0.2.1
...
```

Then I moved the `-` to the line above the comment, ran `yq` again, and got this:
```yaml
entries:
  - # name is required. It is the name of an `olm.bundle` that
    # is present in the channel.  
    name: foo.v0.3.1
    # replaces is optional. It is the name of the bundle that is replaced
    # by this entry. It must be present in the entry list, unless this
    # entry is the channel tail. Channel tails are allowed to have replaces
    # values that are not present in the entry list.
    replaces: foo.v0.2.1
...
```

Which is back to how I have it formatted. I think my preference is to leave it as is, just as `yq` formats it as well.